### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_BlazorServerManageurAccntEmail_NonTextContrast: The color contrast ratio of the focus indicator of the "Change email" button is less than 3:1. 

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42372]
![42372_A11y_ NET Core_ASP NETCore_BlazorServerManageurAccntEmail_NonTextContrast](https://user-images.githubusercontent.com/86710491/175509161-61317eaf-8580-48d9-a53b-5ef05bdfdd6a.jpg)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.11;#DesktopApp;#Win11;#ColorContrast;#Benchmark;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir BlazorServer"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new blazorserver -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. To get access to all the pages you'll need to register first and then log-in.
10. Login with your registered account 
11. Tab till "Hello with the userid" and click on it.
12. Navigate to Email and click on it
13. Turn on Color contrast analyzer and check that The Color contrast ratio of "Change email" is less than 3:1.

### Actual:
The Color contrast ratio of the focus indicator of the "Change email" button is less than 3:1.

### Expected:
The Color contrast ratio of the focus indicator of the  "Change email" button should be more than or equal to 3:1.

### User Impact:
Low vision users won't be able to see the focus indicator.

**Comment by HaoK** [comment]
Should be fixed by https://github.com/dotnet/aspnetcore/pull/43177

**Comment by Truptik12** [comment]
@HaoK 

We downloaded the latest RC1 build from the first column of the above table and verified the bug, and the bug still reproduces.
Please find the attachment of the version.

![image](https://user-images.githubusercontent.com/86710491/185585154-297be56b-9cf1-412d-8625-0ccd8b70aa7c.png)
Could you please provide the repro steps that you followed, if the issue is not reproducing at your end.
.

**Comment by HaoK** [comment]
So there's a bug in the current main builds that prevents blazor server register page from working, but can you verify that you are seeing the expected fix in the `wwwroot/css/site.css` file?

You should see a bunch of different focus i.e. `.btn-link.nav-link:focus, .form-control:focus` defined, can you confirm those are in your newly created app?

**Comment by merriemcgaw** [comment]
@HaoK do you think the bug you referenced is now fixed and @Truptik12 can download the latest?

**Comment by HaoK** [comment]
Yes should be fixed by the css fix

**Comment by Swati1700** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.
![image](https://user-images.githubusercontent.com/86710888/188159638-6fef50ac-0ef6-40cf-8efa-119c908fe4d7.png)

**Comment by Truptik12** [comment]
We have installed latest build dotnet-sdk-7.0.100-rc.2.22452.3-win-x64 but getting a below error.

![image](https://user-images.githubusercontent.com/86710491/188159828-df6c8ebb-2f2a-4ba3-a933-99626a876495.png)

**Comment by merriemcgaw** [comment]
@Truptik12 - the updated machine setup instructions are as follows:
1. On a clean machine install .NET RC2 SDK from the **Top, Middle** cell in the [Installer Table](https://github.com/dotnet/installer#table)
2. Open a command prompt and enter the following command: dotnet nuget add source -n dotnet7 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json
3. Follow the test repro steps from here.

**Comment by Truptik12** [comment]
Verified the bug in the above environment. The bug does not repro. Hence closing the bug. Please find the screenshot attached.

![image](https://user-images.githubusercontent.com/86710491/192525397-343ac449-9f3b-41d3-b1a8-09182fb0cb83.png)

**Comment by mkArtakMSFT** [comment]
Issue is fixed. Adding #Closed; tag.
